src/Lumper.UI/ViewModels/Shared/Pakfile/PakfileLumpViewModel.cs

```diff
@@ -291,6 +291,22 @@ out string[]? opNoExtension
             if (changes > 0)
                 Logger.Info($"Replaced {changes} instances of {updatedOp} with {updatedNp} in file {entry.Key}");
         }
+
+        // TexData - doesn't have viewmodels (yay!)
+        if (oldPath.EndsWith(".vmt", cmp) && opPrefix.Equals("materials", cmp))
```

Comment:
Use `Path.GetExtension` instead.

Reply:
Coming back to this branch and refactoring a bunch of this method + moving it to model code. I don't think `Path.GetExtension` here is appropriate; this code does a lot of very Source-specific path refactoring, not using OS-dependent separators or the likes so `Path` methods aren't applicable in a lot of places. Given this I think it's much cleaner to only using general string methods, not a combination of string methods and Path methods.